sdk/tables/Azure.Data.Tables/samples/Sample1CreateDeleteTables.md

```diff
@@ -15,24 +13,26 @@ var serviceClient = new TableServiceClient(
     new TableSharedKeyCredential(accountName, storageAccountKey));
 ```
 
-## Creating a Table
-To create a table, create a `TableClient` by invoking `CreateTable` from the `TableServiceClient`. A `TableClient` represents a single table and performs table-level operations like inserting and deleting entities within that table. `tableName` can be replaced with any string to be the name of the table. Tables must have unique names.
+## Create a table
+To create a table, invoke `CreateTable` with a [unique table name](https://docs.microsoft.com/en-us/rest/api/storageservices/understanding-the-table-service-data-model#table-names). The `tableName` can be replaced with any string to be the name of the table.
 
 ```C# Snippet:TablesSample1CreateTable
 // Create a new table. The <see cref="TableItem" /> class stores properties of the created table.
 TableItem table = serviceClient.CreateTable(tableName);
 Console.WriteLine($"The created table's name is {table.TableName}.");
 ```
 
-## Deleting a Table
-To delete the table, invoke `DeleteTable` from the service client with the table name.
+A `TableClient` is needed to perform table-level operations like inserting and deleting entities within the table. To get the `TableClient`, invoke `GetTableClient` with the table name.
+
+## Delete a table
+To delete the table, invoke `DeleteTable` with the table name.
 
 ```C# Snippet:TablesSample1DeleteTable
 // Deletes the table made previously.
 serviceClient.DeleteTable(tableName);
```

Comment:
should this ne `tableClient.DeleteTable` instead?

Reply:
No, I was only demonstrating the `TableServiceClient` usage, but I think I might just add in creating/deleting from a `TableClient` to avoid confusion.